src/content/self-hosted/install/releases.mdx

```diff
@@ -25,6 +25,11 @@ id: releases
 - Oketo Self-Hosted will now show the [installation setup guide](administration/dashboard/#installation) by default
 - External resources can now be destroyed from the Okteto UI
 - External resources now supports [additional icons](reference/manifest.mdx##icon-string-optional)
+- Ingresses are now served through two ingress-controllers instances, enabling flexible configuration and improving connection stability.
```

Comment:
Should we mention this in a `Before upgrade` section for customers using their own ingress controller for user applications or for the main ingress controller? They will need to take this into account before upgrade, right?

Reply:
We should mention how to disable both ingress controllers here:
https://www.okteto.com/docs/self-hosted/administration/configuration/#using-your-own-ingress-controller

And add a link to it in the `Before Upgrade` section as @ifbyol suggests